src/services/message.service.ts

```diff
@@ -46,6 +46,13 @@ const sendMessageToDevice = async (
       notification: {
         body: notificationType,
       },
+      apns: {
+        payload: {
+          aps: {
+            'mutable-content': 1,
```

Comment:
Does this work if we set it to `true` rather than `1`?

Reply:
It does!

Reply:
As you know, we confirmed that it does!